### DISCUSSION TITLE
Renovate not able to create an MR properly using yarn-slim

### DISCUSSION THREAD

**Discussion by vladh91** [mend/renovate-ce-ee#278] [Q&A]
We have started to see MR's created by Renovate with the error `No /opt/buildpack/tools/yarn-slim/1.22.19/bin/yarn defined - aborting`
When I check this path directly in renovate pod `yarn-slim` is empty, in the same time `yarn` directory contains binary files. I was wondering why renovate is trying to use `yarn-slim` and not `yarn` and why would `yarn-slim` be empty? 
Please see attached screenshot for the MR error.
![Screenshot 2023-02-08 at 12 04 18](https://user-images.githubusercontent.com/12782143/217470339-3faf21c4-c186-4c89-8121-6aab2c84451f.png)

**Comment by vladh91** [comment]
I found this log and I believe its related

```
WARN: artifactErrors (repository=web-platform/notification-service, branch=renovate/all-minor-patch)
       "artifactErrors": [
         {
           "lockFile": "yarn.lock",
           "stderr": "installing v2 tool yarn-slim v1.22.19\nnpm ERR! code ERR_INVALID_AUTH\nnpm ERR! Invalid auth configuration found: `_auth` must be renamed to `//artifactory-uw2.adobeitc.com/artifactory/api/npm/:_auth` in user config\nnpm ERR! Please run `npm config fix` to repair your configuration.`\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /tmp/tmp.4vIIKh8GpB/_logs/2023-02-07T15_33_57_166Z-debug-0.log\n"
         }
       ]
```

From this log I understood that we have attached npm v6 npmrc configuration but the project itself is using npm v7 and its concerning about the config. Because of it renovate is not able to pull `yarn-slim` package from our private registry. 

In this case is there a way to support 2 versions of npm configs?

**Comment by rarkins** [comment]
Which version of Renovate On-Prem are you running?

**Reply by vladh91** [reply]
Sorry I missed your response. I believe this was already resolved.
